workers/unity/Packages/com.improbable.gdk.ui/DeploymentWindow.cs

```diff
@@ -0,0 +1,186 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Google.LongRunning;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.SpatialOS.Platform.Common;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Tools
+{
+    internal class DeploymentWindow : EditorWindow
+    {
+        private Deployment deployment;
+        private readonly DeploymentServiceClient localDeploymentClient = DeploymentServiceClient.Create(SpatialdEndpoint);
+        private readonly List<Deployment> deployments = new List<Deployment>();
+
+        private const int SpatialDPort = 9876;
+
+        private static readonly PlatformApiEndpoint SpatialdEndpoint = new PlatformApiEndpoint
+        (
+            "localhost",
+            SpatialDPort,
+            true
+        );
+
+        private string projectName;
+
+        private Operation<Deployment, CreateDeploymentMetadata> localDeploymentOperation;
+
+        [MenuItem("Window/SpatialOS/Launch Deployments")]
+        private static void DeploymentWindowMenu()
+        {
+            var window = GetWindow<DeploymentWindow>();
+            window.Show();
+        }
+
+        public void Awake()
+        {
+            titleContent = new GUIContent("Launch deployment");
+        }
+
+        private void OnEnable()
+        {
+            var dict = MiniJSON.Json.Deserialize(File.ReadAllText(Path.Combine(LocalLaunch.SpatialProjectRootDir, "spatialos.json"), Encoding.UTF8));
+            projectName = (string) dict["name"];
+
+            GetDeployments();
+        }
+
+        private async void GetDeployments()
+        {
+            deployments.Clear();
+
+            var nextPageToken = string.Empty;
+            do
+            {
+                var task = localDeploymentClient.ListDeploymentsAsync(new ListDeploymentsRequest
+                {
+                    ProjectName = projectName,
+                    PageToken = nextPageToken,
+                    PageSize = 50
+                });
+
+                var dpls = await task.ReadPageAsync(50);
+                deployments.AddRange(dpls);
+                nextPageToken = dpls.NextPageToken;
+            }
+            while (!string.IsNullOrEmpty(nextPageToken));
+
+            deployment = deployments.FirstOrDefault(dpl =>
+                dpl.Status != Deployment.Types.Status.Stopped);
+        }
+
+        public void OnGUI()
+        {            
+            using (new EditorGUILayout.VerticalScope())
+            {
+                using (new EditorGUILayout.HorizontalScope())
+                {
+                    if (GUILayout.Button("Spatial service log"))
+                    {
+                        if (Application.platform == RuntimePlatform.WindowsEditor)
```

Comment:
Since the other platforms are not used - do we want to print a small warning log? Or are we planning to add this before this PR get merged in?